tests/data_structure/body_model/test_smplxd_data.py

```diff
@@ -181,3 +181,7 @@ def test_file_io():
     with pytest.raises(FileExistsError):
         smplxd_data.dump(npz_path, overwrite=False)
     smplxd_data.dump(npz_path, overwrite=True)
+    # test auto load
+    instance, class_name = auto_load_smpl_data(npz_path)
+    assert isinstance(instance, SMPLXDData)
```

Comment:
Check the instance shape by the way?

Reply:
Fixed in [36c845f](https://github.com/openxrlab/xrmocap/pull/78/commits/36c845f7f7918df025d92827c556e9892a88ddce).